ui/apps/platform/src/Components/CompoundSearchFilter/components/CompoundSearchFilterInputField.tsx

```diff
@@ -58,6 +58,10 @@ function dateParse(date: string): Date {
     );
 }
 
+function getAttributeLabel(name: string): string {
+    return name.toLowerCase().replace(/cve/g, 'CVE');
```

Comment:
Although this is okay for now, what do you think about some brainstorming at next office hours:
* While reviewing contributions from backend engineers, I have noticed that conditional logic for special cases that are not apparent in the configuration objects (like integrations or policy criteria) is a barrier. Maybe even for us :)
* Also for global software, special cases like this need to be in configuration instead of code, whether RegExp or pluralize package.

Reply:
I agree special cases aren't ideal. I was also thinking about the possibility of changing it to something like this:

`Filter results by "Image CVE"`

instead of 

`Filter results by image CVE`

That would forego the need to have a function like this for these special cases. We could simply pass along the value instead of lowercasing it.

Reply:
Oh, yeah.

For extra credit you might ask in forum-acs-docs whether there is any pro and con for use of quotation marks.

If convention for attribute label is Sentence case, then without quotes is readable for me:
Filter results by Image CVE

Reply:
I like your idea. I think without quotations would be better and align with what we already do. Let me try that out 👍🏼